windows/internal/test.bat

```diff
@@ -0,0 +1,39 @@
+@echo off
+
+set SRC_DIR=%~dp0\..
+pushd %SRC_DIR%
+
+set PYTHON_VERSION=%PYTHON_PREFIX:py=cp%
+
+pip install pytest coverage hypothesis protobuf
+
+for /F "delims=" %%i in ('where /R %SRC_DIR%\output\%CUDA_PREFIX% torch*%PYTHON_VERSION%*.whl') do pip install "%%i"
+
+if ERRORLEVEL 1 exit /b 1
+
+echo Smoke testing imports
+python -c "import torch"
+
+echo "Checking that MKL is available"
+python -c "import torch; exit(0 if torch.backends.mkl.is_available() else 1)"
```

Comment:
Will the python exit code be overwritten by subsequent calls?

Reply:
Yes, I'll fix this.

Reply:
Fixed!